src/cell.h

```diff
@@ -141,6 +141,12 @@ struct Cell {
                 p = p == doc->curdrawroot ? nullptr : p->parent;
             if (p) parentcolor = p->actualcellcolor;
         }
+
+        if(sys->_darkennonmatchingcells && !this->text.IsInSearch()) {
+            uchar *cp = (uchar *)&actualcellcolor;
+            loop(i, 4) cp[i] = cp[i] * 800 / 1000;

```

Comment:
This is very slow doing this on a per cell basis. First, this is another check of it being in search, which needs to do a string search. Second, this potentially changes the color of all cells, requiring a lot of extra drawing.

This seems a very niche feature, and I would not agree it should be an option. It's either always on how we display search, or not. I vote for not since its expensive.

Generally, when adding features, consider there is a cost to adding "yet another option" to the UI. The UI should be kept as simple as possible, and only options for when its a thing likely a lot of people will want to interact with.

Reply:
Thanks for your feedback! I stumbled across https://github.com/aardappel/treesheets/issues/200 and thought it would be a nice feature as you also agreed upon in the issue. Should it not be implemented?

Reply:
That is not the only issue, see my comments above.
We can maybe implement it if it can be made efficient (certainly when no search is happening) and likely just always on.

Reply:
Ok, I might come up with a second iteration.